backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala

```diff
@@ -19,16 +19,31 @@ package io.glutenproject.backendsapi.velox
 import io.glutenproject.GlutenConfig
 import io.glutenproject.backendsapi.InitializerApi
 import io.glutenproject.vectorized.{GlutenNativeExpressionEvaluator, JniLibLoader, JniWorkspace}
+import io.glutenproject.utils.{VeloxDllLoaderUbuntu2004,VeloxDllLoaderUbuntu2204,VeloxDllLoader}
 import org.apache.commons.lang3.StringUtils
 import scala.collection.JavaConverters._
+import scala.sys.process._
 
 import org.apache.spark.SparkConf
 
 class VeloxInitializerApi extends InitializerApi {
+  def loadLibFromJar(load: JniLibLoader): Unit = {
+      val system = "cat /etc/os-release".!!
+      if (system.contains("Ubuntu") && system.contains("20.04")) {
+        val loader = new VeloxDllLoaderUbuntu2004
+        loader.loadLib(load)
+      } else if (system.contains("Ubuntu") && system.contains("22.04")) {
+        val loader = new VeloxDllLoaderUbuntu2204
+        loader.loadLib(load)
+      }
```

Comment:
Please try using `System.getProperty("os.name")`

Reply:
Sorry, System.getProperty("os.name") only return "Linux". I had tried it before.
```
public class App {
    public static void main( String[] args ) {
        //Operating system name
        System.out.println(System.getProperty("os.name"));
    }
}
```